public/scripts/customInput.js

```diff
@@ -147,26 +148,26 @@ function handleEnter() {
   }
 
   if (containsErrors) {
-    alert(`sentence has too many errors!`) // TODO better warning
+    alert.alert(`sentence has too many errors!`)
```

Comment:
can you have the `./alert` module export _just_ the `alert` function? I don't think it needs to be wrapped in an object like it is now